### PR TITLE
Closes 1844: We are also not displaying the nutrient modifiers in the product view (< > ~)

### PR DESCRIPTION
**Pull request by brandenfung**
## Description
Added support for displaying modifiers (<, >, ~) for applicable nutrients in the product view (in the Nutrition Tab, Nutrition Facts tab, and the Calculated Nutrition Facts view).

## Related issues and discussion
#1844
 
 ## Screen-shots, if any
imageimageimage

### CODE REVIEW

**Comment by teolemon** [comment]
Whoah :-) 
Thanks @brandenfung 👍

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition_details/NutritionInfoProductFragment.java [hunk 1]
```diff
@@ -176,30 +176,42 @@ public void refreshView(State state) {
...
         // Energy
         Nutriment energy = nutriments.get(ENERGY);
         if (energy != null) {
-            nutrimentItems.add(new NutrimentItem(getString(R.string.nutrition_energy_short_name), Utils.getEnergy(energy.getFor100g()), Utils.getEnergy(energy
-                    .getForServing()), "kcal"));
+            nutrimentItems.add(
+                    new NutrimentItem(getString(R.string.nutrition_energy_short_name),
+                                      Utils.getEnergy(energy.getFor100g()),
+                                      Utils.getEnergy(energy
+                                                              .getForServing()),
```
Comment (huzaifaiftikhar) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition_details/NutritionInfoProductFragment.java]: Fix indentation

Comment (huzaifaiftikhar) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/nutrition_details/NutritionInfoProductFragment.java]: This line can be appended to the above line.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/CalculateDetails.java [hunk 2]
```diff
@@ -98,29 +83,42 @@ protected void onCreate(Bundle savedInstanceState) {
...
         // Carbohydrates
         Nutriments.Nutriment carbohydrates = nutriments.get(CARBOHYDRATES);
         if (carbohydrates != null) {
+            String modifier = nutriments.getModifier(CARBOHYDRATES);
             nutrimentItems.add(new HeaderNutrimentItem(getString(R.string.nutrition_carbohydrate),
-                    carbohydrates.getforanyvalue(value, spinnervalue),
-                    carbohydrates.getForServing(),
-                    carbohydrates.getUnit()));
+                                                       carbohydrates.getforanyvalue(value,
+                                                                                    spinnervalue),
```
Comment (huzaifaiftikhar) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/CalculateDetails.java]: This line can be appended to the above line of code.

**Comment by huzaifaiftikhar** [comment]
Merged, thanks @brandenfung

**Pull request merged**